src/sentry/static/sentry/app/views/settings/organizationTeams/teamMembers.tsx

```diff
@@ -189,19 +198,20 @@ class TeamMembers extends React.Component {
    *
    * @param {Event} e React Event when member filter input changes
    */
-  handleMemberFilterChange = e => {
+  handleMemberFilterChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     this.setState({dropdownBusy: true});
     this.debouncedFetchMembersRequest(e.target.value);
   };
 
-  renderDropdown = access => {
+  renderDropdown = () => {
```

Comment:
this doesn't need to be an arrow function  😉

Reply:
sure :+1: